sycl/source/detail/scheduler/commands.hpp

```diff
@@ -422,8 +422,8 @@ class MemCpyCommand : public Command {
 
   void printDot(std::ostream &Stream) const final;
   const Requirement *getRequirement() const final { return &MDstReq; }
-  void emitInstrumentationData();
-  ContextImplPtr getContext() const override final;
```

Comment:
Why is `override` removed?
AFAIK `final` and `override` are independent.

`getContext()` is `virtual`.

Reply:
FYI: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override